WNPRC_Purchasing/resources/views/purchasingLandingPage.html

```diff
@@ -0,0 +1,25 @@
+<div id="<%=h('wnprc_purchasing-report')%>"></div>
+<script type="text/javascript">
+
+    Ext4.onReady(function()
+    {
+        var webpart = <%=webpartContext%>;
+        var qwp = new LABKEY.QueryWebPart({
+            renderTo: webpart.wrapperDivId,
+            schemaName: 'ehr_purchasing',
+            queryName: 'purchasingRequestsOverview',
+            buttonBar: {
+                includeStandardButtons: true,
```

Comment:
Do we need to explicitly include all of the standard buttons?

Reply:
Yes, otherwise it is listing the custom button first, and didn't see a property to set the location